app-editors/ghostwriter/ghostwriter-1.8.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND="
+	app-text/hunspell
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwebengine:5
+	dev-qt/qtwidgets:5
+"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
```

Comment:
`dev-qt/linguist-tools:5` should be moved to `BDEPEND` array:
`BDEPEND=dev-qt/linguist-tools:5`

Also, are you sure `dev-qt/qtconcurrent:5` is only a build dependency? As of [here](https://github.com/wereturtle/ghostwriter/blob/master/src/HtmlPreview.cpp#L148) in the code, it looks like it is used in runtime. Might be wrong.

Reply:
does not appear as linked lib.